### PR TITLE
FeedbackQuestionDetails: Sanitize question text when displayed #7764

### LINKED ISSUE #7764: FeedbackQuestionDetails: Sanitize question text when displayed
Question text is currently not sanitized when displayed in several instances: eg. on InstructorFeedbackResultsPage and in admin status of InstructorFeedbackQuestionAddAction. It should be sanitized to prevent html rendering issues or script injection.

### PR DESCRIPTION
**Pull request by leeyimin**
Fixes #7764 

**Outline of Solution**

* Sanitized display of question text in statusToAdmin or on html
* Updated tests

### CODE REVIEW

**Comment by leeyimin** [comment]
Placed on hold due to possibly conflicting changes in #7767.

File: src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java [hunk 1]
```diff
@@ -764,6 +767,24 @@ private void testVisibilityOptionsCorrespondToFeedbackPath() {
...
+    private void testSanitization() throws IOException {
+        ______TS("Test sanitization for edit page");
+
+        instructorId = testData.accounts.get("instructor1OfTestingSanitizationCourse").googleId;
+        FeedbackSessionAttributes session = testData.feedbackSessions.get("session1InTestingSanitizationCourse");
+        courseId = session.getCourseId();
+        feedbackSessionName = session.getFeedbackSessionName();
+
+        feedbackEditPage = getFeedbackEditPage();
+        feedbackEditPage.verifyHtmlMainContent("instructorFeedbackEditPageTestingSanitization.html");
```
Comment (thenaesh) [inline src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java]: `/` is missing

File: src/test/resources/data/InstructorFeedbackEditPageUiTest.json [hunk 2]
```diff
@@ -215,9 +252,55 @@
...
+      ],
+      "showGiverNameTo": [
+        "INSTRUCTORS"
+      ],
+      "showRecipientNameTo": [
+        "INSTRUCTORS"
+      ]
     }
   },
-  "feedbackQuestions": {},
```
Comment (thenaesh) [inline src/test/resources/data/InstructorFeedbackEditPageUiTest.json]: ???

Comment (leeyimin) [inline src/test/resources/data/InstructorFeedbackEditPageUiTest.json]: Oops. This change is actually correct, since a feedback question was added.

**Pull request merged**